source/features/add-dashboard-label-links.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+import {wrap} from '../libs/dom-utils';
+import observeEl from '../libs/simplified-element-observer';
+
+function linkifyLabels(activity: HTMLElement): void {
+	const labels = select.all('.IssueLabel', activity);
+	if (labels.length === 0) {
+		return;
+	}
+
+	const repository = select('a[data-hovercard-type="repository"]', activity) as HTMLAnchorElement;
+	if (!repository) {
```

Comment:
Are you sure this check is necessary? Maybe you just need a `!` after `select()` to shut TS up. I expect the repo link to always be there

Reply:
No I also think it should always be there, mainly did it to handle it if it should ever happen.
But I guess it's better to error out then instead of failing silently.